### PR TITLE
feat: New Report Timesheet Billing Summary , combination of three report

### PR DESCRIPTION
**Pull request by viralpatel15**
This Report is Combination of three report

Project Billing Summary
Employee Billing Summary
Daily Timesheet Summary

### CODE REVIEW

File: erpnext/projects/report/timesheet_billing_summary/timesheet_billing_summary.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                    "timesheet": row.get('name'),
+                    "total_billable_hours": total_billing_hours,
+                    "total_hours": total_hours,
+                    "amount": total_amount,
+                }
+            )
+
+    return data
+
+def get_billable_and_total_duration(activity, start_time, end_time):
```
Comment (deepeshgarg007) [inline erpnext/projects/report/timesheet_billing_summary/timesheet_billing_summary.py:215]: An identical function exists in the `billing_summary.py`, that can be used. Why copy paste?

**Review by deepeshgarg007 — CHANGES_REQUESTED** [review]
As you pointed out this is a combination of three existing reports, try and reuse as much code as possible instead of copy-pasting.

Add a screenshot of the report in the description which will help reviewers visualize how the final report looks

**Comment by viralpatel15** [comment]
> As you pointed out this is a combination of three existing reports, try and reuse as much code as possible instead of copy-pasting.
> 
> Add a screenshot of the report in the description which will help reviewers visualize how the final report looks

image

Screenshot 2023-09-25 211326

**Comment by deepeshgarg007** [comment]
@viralpatel15 You can reopen this PR once the requested changes have been made

**Pull request closed without merging**